packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -382,14 +383,17 @@ export type ImplicitFieldSchema = FieldSchema | ImplicitAllowedTypes;
 
 /**
  * Converts ImplicitFieldSchema to the corresponding tree node's field type.
+ * @remarks
+ * When non-exact schema are provided, this errors on the side of returning too general of a type.
+ * This makes this type computation is primarily suited for output APIs where this covariant approach is valid.
```

Comment:
I found this sentence a little confusing as is - maybe something like this is a little more friendly for someone not already working in this space.
```suggestion
 * This is ideal for "output APIs" - i.e. it converts the schema type to the runtime type that a user will _read_ from the tree.
```

Reply:
I think the documentation should say what the API does. It could also say what its ideal for, but it shouldn't document it in terms of what its ideal.

We need to document what it does, so we can tell if the implementation is correct, and if (non-ideal) usages are valid or not. I think this is a good comment to add, but not sufficent.

Reply:
Oh, actually this is just replacing my applications section with a better one. I gotta read more carefully here: though this applies to 2 lines not 1...